Makefile

```diff
@@ -3,7 +3,7 @@ include .env
 
 BINARY := wave
 VERSION := $(shell git describe --always --dirty --tags 2>/dev/null || echo "undefined")
-ECHO := echo
+ECHO := echo -e
```

Comment:
IIRC echo -e doesn't work on all platforms, are you working on linux or mac?

Reply:
Ohh right, yes I recall something similar actually.. (I'm on linux)

If osx has an ancient bash maybe its `echo -e` won't work?

Can you test for me (assuming you're on osx?)
If it doesn't work, Try adding this to the Makefile: `SHELL := /usr/bin/env bash` (to ensure `echo -e` is calling the bash echo)

Otherwise I think we have some options:
1. drop colour prints, instead be clear using old school tricks like: `=========== message =============`
2. I can set the colours via `tput`
3. emojis? ¯\_(ツ)_/¯

I'll give tput a quick test in meantime

Reply:
according to [this](https://stackoverflow.com/questions/28782394/how-to-get-osx-shell-script-to-show-colors-in-echo) `printf` works fine on macOS

So I've updated it and it tests OK on linux (needed to add a `\n` to the end of each statement)

![image](https://user-images.githubusercontent.com/462087/123513195-a6a7b280-d683-11eb-9c55-a3e761f43917.png)